code/modules/antagonists/wraith/critters/voidhound.dm

```diff
@@ -60,3 +60,16 @@
 			boutput(src, "<span class='alert'>We reappear</span>")
 		..()
 
+	Life()
+		var/turf/local_turf = get_turf(src)
+		if (local_turf.RL_GetBrightness() < 0.3 || src.cloaked)
+			if ((src.health < (src.health_brute + src.health_burn)))
+				for(var/damage_type in src.healthlist)
+					var/datum/healthHolder/hh = src.healthlist[damage_type]
+					hh.HealDamage(2)
+			if (!HAS_ATOM_PROPERTY(src, PROP_MOB_STAMINA_REGEN_BONUS))
+				APPLY_ATOM_PROPERTY(src, PROP_MOB_STAMINA_REGEN_BONUS, "voidhound_darkness", 5)
```

Comment:
This would cause the voidhound to not get it's stam regen bonus if it, say, drank a sugary soda first. Intentional?

Reply:
Nope, it's just poor integration. I'll change this to not be terrible.

Reply:
Changed it to a stam status, which is also better for user feedback since they can see it in the top right.